### ISSUE TITLE
the usage of unseen class segmentation annotations

### ISSUE THREAD

**Issue by sunycl** [valeoai/zero_shot_semantic_segmentation#7]
Dear dr.maximebucher:

We would like to thank you for your enlightening paper named "Zero-Shot Semantic Segmentation". Having read your release codes elaborately, we have a kind of confusion about the step 2 training process (In train_pascal_GMMN.py, line 265). The code "loss = self.criterion(output, target)" which means the unseen class segmentation annotations will be inevitably involved in the training procedure of the classifier. In other words, the annotations of the unseen classes shouldn't be used as the supervision, since the zero-shot learning should keep the unseen classes' annotation unavailable. Therefore, we hope to know how the training process of the classifier can achieve the seen and unseen classes recognition. 

Thanks again, wish for your response.

**Comment by maximebucher** [comment]
Hi @sunycl 
Look at #4

**Comment by tuanhungvu** [comment]
https://github.com/valeoai/ZS3/issues/10#issuecomment-740829384
